CRM/Contact/Form/Relationship.php

```diff
@@ -578,22 +578,23 @@ private function createAction($params) {
   /**
    * Prepares parameters to be used for create/update actions
    *
-   * @param array $params
+   * @param array $values
    *
    * @return array
    */
-  private function preparePostProcessParameters($params) {
-    $relationshipTypeParts = explode('_', $params['relationship_type_id']);
+  private function preparePostProcessParameters($values) {
+    $params = $values;
```

Comment:
We need a copy of the `$params` so that we can update `relationship_a_b` and then `relationship_b_a` keys without first overwriting the other.

Reply:
@mattwire is this a php7 v php5 thing?

Reply:
No it's just a computer thing :)
When you change a variable from one value to another, but want to know what value it had before you changed it, you need a 2nd variable.